packages/nx/src/utils/nx-plugin.ts

```diff
@@ -118,7 +118,10 @@ export function readPluginPackageJson(
   json: PackageJson;
 } {
   try {
-    const result = readModulePackageJson(pluginName, paths);
+    const result = readModulePackageJson(pluginName, {
+      requirePaths: paths,
+      noFallback: true,
```

Comment:
Does this not work right now?

Reply:
It logs warnings in node v16+ for local plugins since main.js doesn't exist (its still main.ts). I hadn't noticed before, but during the prompts it writes over the choices until interaction occurs.